plugins/parsers/xml/testcases/cyclehire-london.conf

```
@@ -0,0 +1,34 @@
+# Example for parsing London bicycle for hire station data using the XML parser.
+#
+# File:
+#   testcases/cyclehire-london.xml
+#
+# Expected Output:
+# bikes,address=River\ Street\ \,\ Clerkenwell,host=MBP15-SWANG.local,id=1 installDate=1278947280000,lat=51.52916347,long=-0.109970527,nbBikes=10,nbDocks=19,nbEmptyDocks=9,placement="false",terminalName=1023 1617397861000000000
+# bikes,address=Phillimore\ Gardens\,\ Kensington,host=MBP15-SWANG.local,id=2 installDate=1278585780000,lat=51.49960695,long=-0.197574246,nbBikes=28,nbDocks=37,nbEmptyDocks=9,placement="false",terminalName=1018 1617397861000000000
+# bikes,address=Christopher\ Street\,\ Liverpool\ Street,host=MBP15-SWANG.local,id=3 installDate=1278240360000,lat=51.52128377,long=-0.084605692,nbBikes=2,nbDocks=32,nbEmptyDocks=30,placement="false",terminalName=1012 1617397861000000000
+#
+
+
+
+[[inputs.tail]]
+  files = ["testcases/cyclehire-london.xml"]
+  data_format = "xml"
+
+  [[inputs.tail.xml]]
+    metric_selection = "response/child::location"
+    metric_name = "string('bikes')"
+
+    timestamp = "/stations/@lastUpdate"
+    timestamp_format = "unix_ms"
+
+    field_selection = "child::info"
+    field_name = "name(@*[1])"
+    field_value = "number(@*[1])"
+
+    [inputs.tail.xml.tags]
+      address = "@name"
+      id = "@id"
+
+    [inputs.tail.xml.fields]
+      placement = "string(temporary)"
```

The test-cases (unfortunately) have another format. You should do:
```suggestion

metric_selection = "response/child::location"
metric_name = "string('bikes')"

timestamp = "/stations/@lastUpdate"
timestamp_format = "unix_ms"

field_selection = "child::info"
field_name = "name(@*[1])"
field_value = "number(@*[1])"

[tags]
  address = "@name"
  id = "@id"

[fields]
  placement = "string(temporary)"
```